regression/040_station.nml

```diff
@@ -123,6 +123,15 @@ item (FEAT_STATIONS, basic_station, 255) {
             bitmask(STAT_TILE_NOWIRE),
             bitmask(STAT_TILE_PYLON, STAT_TILE_BLOCKED),
         ];
+        station_layouts: [
+            [
+                [2],
+            ],
+            [
+                [4, 5],
+                [6, 7]
```

Comment:
Should this silently convert to 04 04 06 06 or throw an error?

Reply:
For now it's silently converted, but an error or a warning is possible.